package.json

```diff
@@ -59,8 +60,12 @@
         "doc": "typedoc src && touch docs/.nojekyll",
         "build": "yarn compile",
         "test": "jest test",
-        "lint": "eslint test && eslint src/**/*.ts",
-        "fix": "eslint test --fix && eslint src/**/*.ts --fix",
```

Comment:
Prior commands used in `&&` will short-circuit subsequent commands, i.e. if `eslint test` failed, it wouldn't even run `eslint src`.

Using `concurrently` here guarantees that both will be run.